Admin/AbstractAdmin.php

```diff
@@ -1669,7 +1669,9 @@ public function getSubject()
     {
         if ($this->subject === null && $this->request) {
             $id = $this->request->get($this->getIdParameter());
-            $this->subject = $this->getModelManager()->find($this->class, $id);
+            if (!\is_array($id)) {
```

Comment:
You don't need this backslash, do you? There is no `is_array` method in the current namespace AFAIK